compiler/quilt/tools/core.py

```diff
@@ -74,13 +79,102 @@ def __json__(self):
         val = super(GroupNode, self).__json__()
         val['children'] = self.children
         return val
-
+   
 
 class RootNode(GroupNode):
     __slots__ = ()
 
     json_type = 'ROOT'
 
+    def __getitem__(self, item):
```

Comment:
I'd rather just get rid of this. I don't know why it was added to Package in the first place, it was never a public API, and supporting the "/" syntax just feels weird to me.

Reply:
I agree, but it's used in build. Can we do that in a separate PR?

Reply:
Nevermind. I removed those and all changes to core.py.